js/src/blob.js

```diff
@@ -39,6 +41,14 @@ export default class Blob extends Collection<IndexedSequence> {
   get length(): number {
     return this.sequence.numLeaves;
   }
+
+  splice(idx: number, deleteCount: number, ...insert: number[]): Promise<Blob> {
```

Comment:
I'm not sure we want `...insert: number[]` here. Maybe we should pass a `Uint8Array` here instead.